src/onesignal.ts

```diff
@@ -81,9 +86,38 @@ export async function sendReadingReminderPush(
     'push',
     'reminder',
   );
-  push.chrome_web_badge =
-    'https://daily-now-res.cloudinary.com/image/upload/v1672745846/public/dailydev.png';
+  push.chrome_web_badge = chromeWebBadge;
   push.chrome_web_icon =
     'https://daily-now-res.cloudinary.com/image/upload/s--9vc188bS--/f_auto/v1712221649/1_smcxpz';
   await client.createNotification(push);
 }
+
+export async function sendStreakReminderPush(
+  userIds: string[],
+  at: Date,
+): Promise<null | OneSignal.CreateNotificationSuccessResponse> {
+  if (!appId || !apiKey) return null;
+  const push = new OneSignal.Notification();
+  push.app_id = appId;
+  push.include_external_user_ids = userIds;
+  push.send_after = at.toISOString();
+  push.contents = {
+    en: streakReminderContent,
+  };
+  push.headings = {
+    en: streakReminderHeading,
+  };
+  push.url = addNotificationUtm(
+    process.env.COMMENTS_PREFIX,
+    'push',
+    // @TODO: check what this should be
+    'streak_reminder',
+  );
```

Comment:
@idoshamun @rebelchris what should this value be? Is `streak_reminder` ok?

Example URL `http://webapp.local.com:5002/?utm_source=notification&utm_medium=push&utm_campaign=streak_reminder`

Reply:
From my side it's ok :)

Reply:
```suggestion
  push.url = addNotificationUtm(
    process.env.COMMENTS_PREFIX,
    'push',
    'streak_reminder',
  );
```

Reply:
yap sounds good